crates/ruff/src/fs.rs

```diff
@@ -88,12 +86,3 @@ pub fn relativize_path(path: impl AsRef<Path>) -> String {
     }
     format!("{}", path.display())
 }
-
-/// Read a file's contents from disk.
-pub fn read_file<P: AsRef<Path>>(path: P) -> Result<String> {
```

Comment:
This fn is identical to `std::fs::read_to_string`

Reply:
Lol, I hope it either _was_ different at one point, _or_ it was just one of the first few functions I wrote and I didn't realize they were the same.